### ISSUE TITLE
Add more options for maker:controller

### ISSUE THREAD

**Issue by zelf** [symfony/maker-bundle#109]
Hey guys, 
Currently, unless I missed the option somewhere, when I do
```sh
$ bin/console make:con

 Choose a name for your controller class (e.g. OrangeKangarooController):
 > NameController
```

It create a controller using annotations.

```php
/**
 * @Route("/name", name="name")
 */
```

What do you think of asking for the desired route name, path and the strategy:
```sh
$ bin/console make:controller

 Choose a name for your controller class (e.g. OrangeKangarooController):
 > NameController

Choose a path (default: name)
> some-path

Choose a route name (default: name)
> some-name

Annotations or yaml? (default: Annotations)
> yaml
```

If you think this could be an interesting feature, I could give it a try.
Cheers.

**Comment by javiereguiluz** [comment]
@zelf I appreciate that you propose new features to this bundle but I'm 👎 on this feature.

When we designed this bundle, we decided to make some decisions to avoid the complexity of the old SensioGeneratorBundle. For example: your application must use Flex, it must also use the dir structure recommended by Symfony 4, etc. Using annotations for routes is one of those simplifications that make this bundle possible. I hope you understand it. Thanks!
